### PR TITLE
Dev/10: Exam List page

### PR DESCRIPTION
**Pull request by Johnny850807**
Complete #10 
image

### CODE REVIEW

File: src/app/App.js [hunk 1]
```diff
@@ -1,18 +1,22 @@
 import './App.css';
 import * as React from "react";
-import 'bulma';
 import {BrowserRouter as Router, Route} from "react-router-dom";
 import {Login} from "./Login";
 import {Dashboard} from "./Dashboard";
 import {ProblemEditor} from "./problem/ProblemEditor";
+import 'bulma';
+import 'bulma';
```
Comment (Chaoyuuu) [inline src/app/App.js:8]: double `bluma` ??

File: src/app/exam/ExamList.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
+.container {
+    padding-top: 60px;
+    padding-left: 100px;
+    padding-right: 100px;
+}
+
+.custom-title-font {
+    font-size: 35px;
+    font-weight: bold;
```
Comment (Chaoyuuu) [inline src/app/exam/ExamList.css:9]: change to `font-weight: 800`

**Review by Chaoyuuu — CHANGES_REQUESTED** [review]
Review Done 😄

File: src/app/exam/ExamList.css (hunk 2 above)
Comment (Johnny850807) [reply src/app/exam/ExamList.css:9]: it can't increase :\

**Review by Chaoyuuu — APPROVED** [review]
LGTM

**Pull request merged**